src/core/lib/surface/legacy_channel.cc

```diff
@@ -93,10 +93,8 @@ absl::StatusOr<OrphanablePtr<Channel>> LegacyChannel::Create(
     *(*r)->stats_plugin_group =
         GlobalStatsPluginRegistry::GetStatsPluginsForServer(args);
   } else {
-    // TODO(roth): Figure out how to populate authority here.
-    // Or maybe just don't worry about this if no one needs it until after
-    // the call v3 stack lands.
-    StatsPlugin::ChannelScope scope(target, "");
+    StatsPlugin::ChannelScope scope(
+        target, args.GetOwnedString(GRPC_ARG_DEFAULT_AUTHORITY).value_or(""));
```

Comment:
This isn't going to work as you expected, because in the majority of cases that channel arg is not actually set until much lower in the stack.  And unfortunately, it's not trivial to fix this in the current channel stack, although it will be easier in the new v3 stack.  That's why I had the TODO here to begin with.

For now, how about we simply remove the authority field from `ChannelScope`, since we don't actually need it right now?

Reply:
I know that the `client_channel_filter` and the resolvers can override the authority to be used, and if there is no override, we derive the authority from the target.

This is exactly why I'm wording it as "Returns the default authority set for the channel, empty if un-set". I could be specific and say that it only shows the default authority arg added by the application, but I want some wiggle room. Also, a reason why we should probably slap the EXPERIMENTAL label here.

Reply:
done as discussed offline